asignment/lib/listview.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'model.dart';
+import 'package:provider/provider.dart';
+
+class ItemList extends StatefulWidget {
+  final List<Item> list;
+  ItemList(this.list);
+
+  @override
+  _ItemListState createState() => _ItemListState();
+}
+
+// Kollar igenom listan och bygger den
+class _ItemListState extends State<ItemList> {
+  @override
+  Widget build(BuildContext context) {
+    return ListView.builder(
+        itemCount: widget.list.length,
+        itemBuilder: (context, index) => _item(widget.list[index]));
+  }
+
+  Widget _item(Item item) {
+    return ListTile(
+      leading: Checkbox(
+        value: item.isDone,
+        onChanged: (bool value) {
+          setState(() {
+            item.isDone = value;
+          });
+        },
+      ),
+      title: Text(item.name, style: TextStyle(color: Colors.black)),
```

Comment:
Här kan du lägga in "decoration:" vid color: te.x. och kolla om item.isDone == true, så kan du få in en TextDecoration.lineThrough som kommer över din todo när den är i klickad av checkboxen :) 👍